.circleci/config.yml

```diff
@@ -34,6 +34,11 @@ jobs:
           command: |
             node -v
             bb ci:test
+      - run:
+          name: Run build tests
+          command: |
+            node -v
+            bb ci:test-build
```

Comment:
I thought you might want this a separate job since it's a little long than the average test with the custom release build